src/cdk/stepper/stepper.ts

```diff
@@ -243,7 +244,7 @@ export class CdkStepper implements AfterViewInit, OnDestroy {
   @ContentChildren(CdkStep) _steps: QueryList<CdkStep>;
 
   /** The list of step headers of the steps in the stepper. */
-  _stepHeader: QueryList<FocusableOption>;
+  @ContentChildren(CdkStepHeader) _stepHeader: QueryList<CdkStepHeader>;
```

Comment:
@crisbeto this is technically a breaking change right now, can you change it to this instead:

```ts
@ContentChildren(CdkStepHeader) _stepHeader: QueryList<FocusableOption>;
```

Reply:
Done. I didn't consider it a breaking change, because the property is underscored.